src/group_operations/addition_operation.jl

```diff
@@ -148,8 +148,13 @@ function identity_element(
 ) where {𝔽,T<:Number}
     return fill(zero(T))
 end
-function identity_element(::LieGroup{𝔽,AdditionGroupOperation}, e::Number) where {𝔽}
-    return zero(e)
+function identity_element(::LieGroup{𝔽,AdditionGroupOperation}, T::Type{<:Number}) where {𝔽}
```

Comment:
This might already be covered by the definition in line 141 though?

Reply:
(maybe missing the `<:` there ;) )

Reply:
I didn't notice that line somehow. I've updated the code and fixed another issue in it :slightly_smiling_face:

Reply:
No problem, that is exactly what code review is for :) I mainly noticed, that line 141 was no longer covered by code cov.